tests/testthat/test_remove_funs.R

```diff
@@ -19,18 +19,28 @@ context("halveString")
 
 ###
 test_that("halveString", {
-  str <- c("a__b__c", "a___m___k__")
-  d1  <- halveString(str, pattern = "__", colnames= c("a", "b"))
-  d2  <- halveString(str, pattern = "__", first=FALSE)
-  expect_equal(class(d1), c("matrix", "array"))
-  expect_equal(colnames(d1), c("a", "b"))
-  expect_equal(colnames(d2), c("X1", "X2"))
-  expect_equal(dim(d2), c(2,2))
-  expect_equal(dim(d1), c(2,2))
-  expect_equal(d1[,1], rep("a", 2))
-  expect_equal(d1[,2], c("b__c", "_m___k__"))
-  expect_equal(d2[,1], c("a__b", "a___m__"))
-  expect_equal(d2[,2], c("c", "_k"))
+  str1<- c("a__b__c", "a___m___k__")
```

Comment:
LGTM
Nits: Remove `testthat::` from all function calls and split test_that("halve_string", ...) into multiple, meaningful `test_that()` calls.

Reply:
done ...